### PR TITLE
don't invent backends in worker settings

### PR DESCRIPTION
**Pull request by coolo**

### CODE REVIEW

**Comment by coolo** [comment]
this should most likely not be merged, but redone

**Comment by coolo** [comment]
now with changed schema

**Comment by aaannz** [comment]
LGTM.
What about the one failed travis? Seems to me the travis was too slow and the test hit the dead worker detection. This raises concern the failure will repeat itself triggering false positives.

**Comment by coolo** [comment]
so either we increase the timeout or accept dead too

**Comment by coolo** [comment]
see bb20d99

File: dbicdh/SQLite/upgrade/27-28/001-auto.sql [hunk 1]
```diff
@@ -0,0 +1,47 @@
+-- Convert schema '/home/coolo/prod/openQA/script/../dbicdh/_source/deploy/27/001-auto.yml' to '/home/coolo/prod/openQA/script/../dbicdh/_source/deploy/28/001-auto.yml':;
+
+;
+BEGIN;
+
+;
+DROP INDEX jobs_fk_worker_id;
```
Comment (aaannz) [inline dbicdh/SQLite/upgrade/27-28/001-auto.sql]: This makes the SQLite migration fail during package upgrade

**Pull request merged**